crates/criticaltrust/src/errors.rs

```diff
@@ -23,6 +26,14 @@ pub enum Error {
     InvalidKey(String),
     #[error("unsupported key")]
     UnsupportedKey,
+    #[error("verification failed because the signatures expired")]
+    SignaturesExpired,
+    #[error("revocation info is expired")]
```

Comment:
Can we specify when it expired so if they contact us we can more easily debug (it may also hint to them that their local clock may be wrong)

Reply:
That SignaturesExpired is old code :see_no_evil:. I removed it. Adding that to the actual Error enum variant to show the date time.

Reply:
```
error: failed to run install command
  caused by: verification failed because the signatures expired on '2024-10-01 +00::00'
```